samples/twitter-timeline/extension/index.ts

```diff
@@ -0,0 +1,30 @@
+import { NodeCG } from "nodecg/types/server";
+import { ServiceProvider } from "nodecg-io-core/extension/types";
+import { TwitterServiceClient } from "nodecg-io-twitter/extension";
+
+module.exports = function (nodecg: NodeCG) {
+    nodecg.log.info("Sample bundle for twitter started");
+
+    // This explicit cast determines the client type in the requireService call
+    const twitter = (nodecg.extensions["nodecg-io-twitter"] as unknown) as
+        | ServiceProvider<TwitterServiceClient>
+        | undefined;
+
+    twitter?.requireService(
+        "twitter-timeline",
+        (client) => {
+            nodecg.log.info("Twitch client has been updated, adding handlers for messages.");
+            const twitterClient = client.getRawClient();
+            const params = {
+                screen_name: "skate702", // eslint-disable-line camelcase
+                exclude_replies: true, // eslint-disable-line camelcase
+                count: 50,
+            };
+            twitterClient
+                .get("statuses/user_timeline", params)
+                .then((tweets: any[]) => tweets.forEach((tweet: any) => nodecg.log.info(`Got tweet: ${tweet.text}`)))
+                .catch((err: any) => nodecg.log.error(err));
```

Comment:
Some types can be auto-infered here and we have to ignore the last any, as we don't have any type information.
```suggestion
                // eslint-disable-next-line @typescript-eslint/no-explicit-any
                .then((tweets: any[]) => tweets.forEach((tweet) => nodecg.log.info(`Got tweet: ${tweet.text}`)))
                .catch((err) => nodecg.log.error(err));
```

Reply:
In theory this would be of type _object_, but _object_ typings in ts aren't that great either

Reply:
https://developer.twitter.com/en/docs/tweets/timelines/api-reference/get-statuses-user_timeline